src/nyc_trees/apps/home/views.py

```diff
@@ -51,10 +51,11 @@ def _global_counts(past_week=False):
     else:
         blocks_mapped = Survey.objects.all().distinct('blockface_id').count()
     if blocks_mapped > 0:
-        blocks_percent = "{:.0%}".format(
+        fmt = "{:.0%}" if blocks_mapped >= 10 else "{:.1%}"
```

Comment:
Should this check for percentage of blocks mapped instead of number of blocks mapped?